listener/inventory_processor.py

```diff
@@ -193,16 +194,20 @@ async def _db_import_repos(self, conn, repo_list: list) -> [str]:
                     if row["inserted"]:
                         NEW_REPO.inc()
                         repo_names.append(row["name"])
-                    self.repo_id_cache[row["name"]] = row["id"]
-        return repo_names
+                    repo_id_mapping[row["name"]] = row["id"]
+        return repo_names, repo_id_mapping
```

Comment:
since the `repo_names` isn't used anywhere, let's refactor it further and remove it

and return only `repo_id_mapping`